### PR TITLE
I can't believe I did this

### PR DESCRIPTION
**Pull request by SticksDev**

### CODE REVIEW

**Comment by danii** [comment]
Hi, it's been a while, I've finally bought enough salt to fend off the demons that'll come after I merge this, so let's review.

File: src/lib.rs [hunk 1]
```diff
@@ -37,43 +37,44 @@ fn iterate_tokens(tokens: TokenStream) -> TokenStream {
...
+		wetpussy => let,
 		hecc => loop,
 		sex => match,
 		discowdmodewatow => mod,
 		movemedaddy => move,
-		mutt => mut,
+		makemeadog => mut,
 		pubes => pub,
-		wef => ref,
+		wef => ref,whinebecauseyouhurtmelastnightsad
```
Comment (danii) [inline src/lib.rs:65]: Appears you've made a mistake here.

File: src/lib.rs [hunk 2]
```diff
@@ -37,43 +37,44 @@ fn iterate_tokens(tokens: TokenStream) -> TokenStream {
 
 fn parse_token(token: TokenTree) -> TokenTree {
 	replace_keywords!(token {
-		Me_Irl => Self,
-		ass => as,
+		//! This made me cry to do -sticks
+		me_iww => Self,
+		boipussy  => as,
```
Comment (danii) [inline src/lib.rs:42]: Are you absolutely sure this follows [GitHub's guidelines](https://docs.github.com/en/github/site-policy/github-community-guidelines#sexually-obscene-content)? I understand this whole project is a joke, but I don't want any of us breaking any rules here.

**Review by danii — CHANGES_REQUESTED** [review]

**Comment by AlekEagle** [comment]
**YELLING**

**Comment by WaviestBalloon** [comment]
Yeller -> Yellow

**Comment by WaviestBalloon** [comment]
Wow.....

**Comment by SticksDev** [comment]
LMAO..

I did a repo clean ok lol

**Pull request closed without merging**